flink-runtime/src/test/java/org/apache/flink/runtime/checkpoint/channel/SequentialChannelStateReaderImplTest.java

```diff
@@ -184,7 +186,7 @@ private Map<InputChannelInfo, List<Buffer>> collectBuffers(InputGate[] gates) th
     private void assertConsumed(InputGate[] gates)
             throws InterruptedException, java.util.concurrent.ExecutionException {
         for (InputGate gate : gates) {
-            assertTrue(gate.getStateConsumedFuture().isDone());
+            assertThat(gate.getStateConsumedFuture().isDone()).isTrue();
```

Comment:
```suggestion
            assertThat(gate.getStateConsumedFuture()).isDone();
```
assertJ supports some `CompletableFuture` methods